barcamp-theme/templates/base.html

```diff
@@ -61,7 +61,11 @@
 
 		    <p class="lead">Beaver BarCamp is an informal conference where everyone is encouraged to participate and the sessions are not predetermined. <a href="/pages/about.html"><em>Learn more</em></a></p>-->
 		    <h5>Event Sponsors</h5>
-        <p>Want to sponsor the event? Please contact info < at > beaverbarcamp.org</p>
+            <img class='sponsor' src='/theme/img/github.png' />
```

Comment:
Oh?  The information sheet for BBC15 was where I got the list from... 
https://docs.google.com/a/osuosl.org/document/d/1CWtkoamNHjX_515ux1EvVT__i7_kdW_DHxeZap41vDQ/edit